docs/model_developers/modules.rst

```diff
@@ -329,3 +329,61 @@ for supplying the agents with the appropriate ``inmanta_plugins`` packages.
 The only exception to this rule is when using the ``inmanta export`` command. It exports a project and all its modules'
 ``inmanta_plugins`` packages to the orchestrator server. When this method is used, the orchestrator does not install any modules
 from the Python package repository but instead contains all Python code as present in the local Python environment.
+
+Configure a project to install modules from a private python package repository
+--------------------------------------------------------------------------------
+
+V2 modules are Python packages that can be installed from private Python package repositories, which may require authentication.This section explains the steps to install such modules.
+
+Create a file named ``/var/lib/inmanta/.netrc`` in the orchestrator's file system.
+Add the following content to the file:
+
+.. code-block:: text
+
+  machine <url of the private repository>
+  login token
+  password <the token to access the private repository>
+
+Next choose one of the following two options to specify to the inmanta server where to get the modules:
+
+1. Use the pip config file
```

Comment:
Now that I think about it, the two section mentioned below describe the difference between 1) using a index URL defined in the global pip config and 2) using a index URL defined in the project.yml file. This is completely unrelated to whether or not the Python package repository requires authentication. I see that it's interesting to mention this information somewhere in the documentation, but I don't think it should be part of this section.

Reply:
I changed the location but i'm still not sure it is the best place. what do you think?